### ISSUE TITLE
Standalone example?

### ISSUE THREAD

**Issue by henryiii** [Quansight-Labs/free-threaded-compatibility#16]
Would a small standalone example (or more) be a good idea? I just wrote one at https://github.com/scikit-build/scikit-build-sample-projects/pull/50/files, which does a simple pi calculation in pure Python and with a C extension (with a little C++ for the random number generator) and a scikit-build-core build. Doing so was a bit tricky, as docs are currently scattered or in PEPs (multiphase initialization seems to be mostly in it's PEP).

**Comment by lysnikolaou** [comment]
Agreed. I'm not sure whether this is the correct place to *host* such tutorials/guides/articles, but we certainly should be linking to resources about all of this from our README, or even maybe a new resources doc.
